server/controllers/report.controller.js

```diff
@@ -45,8 +45,9 @@ function create(req, res, next) {
  */
 function update(req, res, next) {
   const report = req.report;
-  report.urls = req.body.urls;
-  report.standard = req.body.standard;
+  report.urls = req.body.urls || report.urls;
```

Comment:
Lets you specify multiple fields to update or only 1 (useful when updating progress)